client/src/pages/Checkout.js

```diff
@@ -8,52 +26,126 @@ import {
   useElements,
 } from "@stripe/react-stripe-js";
 import { loadStripe } from "@stripe/stripe-js";
-import { createPaymentIntent } from "../services";
+
+import { createPaymentIntent, confirmPaymentIntent } from "../services";
+import loadingContext from "../loadingContext";
+const STRIPE_API_KEY =
+  "pk_test_51H9aiQHBjJBD671jjlNd2YSaGTgUaRtAfrSOlF1271z4ftGZwKSP2UVI0XWHuQqzA5NAY7pv5FvuQj1LT47nokHG00DZWM1eoW";
```

Comment:
I'd probably load this using this - https://create-react-app.dev/docs/adding-custom-environment-variables/ - to make it cleaner